action/completion.go

```diff
@@ -51,3 +51,33 @@ source <(gopass completion bash)
 
 	return nil
 }
+
+// CompletionDMenu returns a script that starts dmenu
+// Usage: eval "$(pass completion dmenu)"
+func (s *Action) CompletionDMenu(
+	c *cli.Context) error {
+	out := `#!/usr/bin/env bash
+
+shopt -s nullglob globstar
+
+typeit=0
+if [[ $1 == "--type" ]]; then
+	typeit=1
+	shift
+fi
+
+password=$(gopass list --raw | dmenu "$@")
+
+[[ -n $password ]] || exit
+
+if [[ $typeit -eq 0 ]]; then
+	pass show -c "$password" 2>/dev/null
```

Comment:
Nope, that's by mistake and worked for me because I `alias pass=gopass`... :smile: Thankfully still WIP and will be update, thanks for pointing out anyway! :blush: